tendrl/gluster_integration/objects/gluster_brick/atoms/create/__init__.py

```diff
@@ -0,0 +1,105 @@
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons import objects
+from tendrl.gluster_integration.objects.gluster_brick import GlusterBrick
+
+
+class Create(objects.BaseAtom):
+    obj = GlusterBrick
+    def __init__(self, *args, **kwargs):
+        super(Create, self).__init__(*args, **kwargs)
+
+    def run(self):
+        bricks = self.parameters.get('GlusterBrick.brick_dict')
+        brick_dict = {}
+        for k, v in bricks.iteritems():
+            host = k.split(":")[0]
+            if not brick_dict.has_key(host):
+                brick_dict[host] = {}
+            disk = v["disk"]
+            brick_path = k.split(":")[-1]
+            mount_path = "/" + "/".join(brick_path.split("/")[1:-1])
+            brick_dict[host].update({
+                disk: {
+                    "node_id": v["node_id"],
+                    "mount_path": mount_path,
+                    "brick_path": brick_path,
+                    "lv": "tendrl" + brick_path.replace("/", "_") + "_lv",
```

Comment:
Here, tendrl is making several conventions regarding naming. As discussed above, such conventions need to be applied to standardise a mount path where tendrl would be able to pick up the available bricks from. Further implication is that the sync operation needs to scan the mount path and gather the mapping of available bricks their corresponding devices.